gramps/gui/views/treemodels/peoplemodel.py

```diff
@@ -322,7 +324,8 @@ def _get_death_data(self, data, sort_mode):
             event = self.db.get_event_from_handle(er.ref)
             etype = event.get_type()
             date_str = get_date(event)
-            if (etype in [EventType.BURIAL,
+            if (etype in [EventType.STILLBIRTH,
```

Comment:
Use `etype.is_death_fallback()` instead.

Reply:
The same issue appeared at line 407

Fixed in [15a0085](https://github.com/gramps-project/gramps/pull/1148/commits/15a0085b6316bb277400a9133eabf32b47be2558)